### DISCUSSION TITLE
有没有人救救我的邦布啊！！！

### DISCUSSION THREAD

**Discussion by lmingzhe86** [share121/inter-knot#4690] [绝区零]
我在空洞被以骸袭击了！我的邦布小波奇为了救我帮我吸引走了以骸    现在我联系不到她！！！
有没有绳匠可以帮帮我QWQ

**Comment by lmingzhe86** [comment]
这个委托我接了！

**Reply by sensei-is-playing-GenShin** [reply]
禁止自娱自乐（doge）

**Reply by lmingzhe86** [reply]
喵。

**Comment by tingyu919** [comment]
承接委托（进空洞的不接）（狗头）

**Reply by lmingzhe86** [reply]
![9536cff62fcfbc56480e4e5cc2463eb1a27ed61b](https://github.com/user-attachments/assets/b25151a8-5e7f-4d91-9069-b3546508eaf6)

**Comment by ZZZAnBi** [comment]
这个委托我不接了

**Reply by lmingzhe86** [reply]
补药啊
